azkaban-common/src/main/java/azkaban/executor/container/watch/FlowStatusManagerListener.java

```diff
@@ -132,7 +132,7 @@ private void validateAndPreProcess(AzPodStatusMetadata event) {
               event.getPodName()));
       logger.error("Unsupported state transition.", transitionException);
       try {
-        finalizeFlowAndDeleteContainer(event);
+        finalizeFlowAndDeleteContainer(event, Optional.of(Status.EXECUTION_STOPPED));
```

Comment:
In the case of Pod going down due to OOM caused by user code. Will the execution be marked with status FAILED or EXECUTION_STOPPED? @sshardool @jakhani @djaiswal83 @aditya1105